torch/export/_trace.py

```diff
@@ -277,24 +278,32 @@ def _extract_fake_inputs(gm, args, kwargs):
         else:
             fake_vals.append(node.meta.get("example_value"))
 
-    if in_shuffle_graph := getattr(gm, "_in_shuffle_graph", None):
-        flat_args = pytree.tree_leaves((args, kwargs))
-        node_map = {
-            node: i
-            for i, node in enumerate(
-                next(iter(reversed(in_shuffle_graph.graph.nodes))).args[0]
-            )
-            if node.op == "placeholder"
-        }
-        new_fake_inps: list[Any] = []
-        for i, node in enumerate(
-            in_shuffle_graph.graph.find_nodes(op="placeholder")[1:]
-        ):
-            if node in node_map:
-                new_fake_inps.append(fake_inps[node_map[node]])
+    if dynamo_bytecode_flatten := getattr(gm, "_dynamo_bytecode_flatten", None):
```

Comment:
Wait why do we even need this? Can't we just use fake mode from dynamo directly refakify the inputs? So that we can avoid this whole block?

Reply:
> Can't we just use fake mode from dynamo directly refakify the inputs? So that we can avoid this whole block?

Again, fake tensor can be handled this way, but this approach doesn't scale to symint inputs.